src/ImageSharp/Memory/Allocators/MemoryAllocatorOptions.cs

```diff
@@ -27,4 +28,22 @@ public struct MemoryAllocatorOptions
             this.maximumPoolSizeMegabytes = value;
         }
     }
+
+    /// <summary>
+    /// Gets or sets a value defining the maximum (discontiguous) buffer size that can be allocated by the allocator in Megabytes.
+    /// <see langword="null"/> means platform default: 1GB on 32-bit processes, 4GB on 64-bit processes.
+    /// </summary>
+    public int? AllocationLimitMegabytes
```

Comment:
Is this consistent with how we talk about memory across the public api? do we talk about megabytes anywhere else in the public api or is it all bytes?

we want to make sure we are being consistent on our units of measure in the public api surface.

Reply:
The other property on `MemoryAllocatiorOptions` is `MaximumPoolSizeMegabytes`. When I designed the [MemoryAllocator configuration API](https://docs.sixlabors.com/articles/imagesharp/memorymanagement.html#configuring-the-pool-size) it seemed like Megabytes are easier to reason about when talking about limits.